src/library_glut.js

```diff
@@ -361,15 +367,11 @@ var LibraryGLUT = {
     window.addEventListener("mousedown", GLUT.onMouseButtonDown, true);
     window.addEventListener("mouseup", GLUT.onMouseButtonUp, true);
     // IE9, Chrome, Safari, Opera
-    window.addEventListener("mousewheel", GLUT.onMouseWheel, true);
+    window.addEventListener("mousewheel", GLUT.onMouseWheel, { passive: false });
```

Comment:
this changes the capture property, doesn't it? shouldn't we add `capture: true` to be backwards compatible? or is this part of the fix?

Reply:
You are right. The "passive: false" change is not part of the fix. This is just to silence Chrome's warning about passive event listeners trying to preventDefault.